src/main.cpp

```diff
@@ -235,11 +241,31 @@ int main(int argc, char **argv) {
 		return EXIT_FAILURE;
 	}
 
+   PlayerInputComponent* playerInputComp = new PlayerInputComponent();

```

Comment:
Hmm yeah, not sure, maybe create within the camera constructor then? We'll probably have a Camera base class and then sub-classes at some point to support things like the "preview cam" so it makes sense to me to keep player logic local to the "player cam".  Plus keeps more stuff out of main.

Can leave where it is for now though if you want.